roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Install Apache httpd
+  ansible.builtin.package:
+    name:
+      - httpd
+      - mod_ssl
+    state: present
+
+- name: Set httpd_can_network_connect so Apache can connect to Puma and Gunicorn
+  ansible.posix.seboolean:
+    name: httpd_can_network_connect
+    state: true
+    persistent: true
+
+- name: Disable welcome page
+  ansible.builtin.file:
+    path: /etc/httpd/conf.d/welcome.conf
+    state: absent
+
+- name: Create cert directories
+  ansible.builtin.file:
+    path: "{{ item }}"
+    state: directory
+  loop:
+    - /etc/pki/katello/certs
+    - /etc/pki/katello/private
```

Comment:
I'm OK with this for parity, but I would love to change this location.
I suppose if we were to run Apache as a container these would be secrets and it'd be a moot point.

Reply:
Tell me a good location and I'll move :)